source/process/release-process.md

```diff
@@ -110,8 +110,9 @@ Exceptions can be made by the release manager setting priority to "Highest" and
     - (Logistics) Walk through each unfinished item of this checklist  
     - (Dev) Last check of tickets that need to be merged before RC1  
 5. Build:  
-    - Review all `XXX FIXME` notes 
+    - Review all `TODO` notes 
     - Master is tagged and branched and “Release Candidate 1″ is cut (e.g. 1.1.0-RC1) according to the Release Candidate Checklist in ``mattermost/process``
+    - After branching, the database version in sql_upgrade.go on master is set to the following version
```

Comment:
Hey @hmhealey, I'm not really sure what "set to the following version" means, can you please make this more clear?